views/home/index.php

```diff
@@ -343,7 +340,3 @@ functionality in this new engine for PHP Xdebug wouldn't compile anymore, with
 
 <!-- MAIN FEATURE END -->
 
```

Comment:
You can remove this whole MAIN FEATURE END line too - and I guess the `<br /><br />`'s — but I suppose layout is for a next pass?

Reply:
yeah, next pass will clean up a bunch of stuff (including tables)